### PR TITLE
chore: separate issue_templates into categories

### PR DESCRIPTION
**Pull request by SoaresMG**
# PR Details

Added the following Issue Templates:

1. **Bug** - As the name says it's for bug reporting
2. **Feature** - New feature/improvement
3. **Other** - Neither 1) or 2)

Also added `Chore` to the PR template.

## Description

It was a bit confusing when adding a new issue since the developers didn't know which sections of the template they could remove.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Chore (workflows / linting / tooling)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](../docs/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)

### CODE REVIEW

File: .github/workflows/ISSUE_TEMPLATE/other.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Issue Template
+about: Other issues template.
+title: ''
+labels: ''
```
Comment (joaopedrodcf) [inline .github/workflows/ISSUE_TEMPLATE/other.md:5]: Maybe we could have a label other just to faster parse this types of issues?

Comment (SoaresMG) [reply .github/workflows/ISSUE_TEMPLATE/other.md:5]: Not sure if that's really needed, at least for now we are not having difficulties managing new issues.

Let's evaluate that in the future

Comment (joaopedrodcf) [reply .github/workflows/ISSUE_TEMPLATE/other.md:5]: Sure 👍

**Pull request merged**